src/proper_parents.py

```diff
@@ -0,0 +1,19 @@
+"""This will tell you if your string is all messed up."""
+
+
+def pp(string):
```

Comment:
Use better variable names. I don't immediately know what `pp` is supposed to do. Also this function could use its own doc string to describe what it's supposed to be accomplishing